recipe/migrations/cuda120_ppc64le_aarch64.yaml

```diff
@@ -0,0 +1,48 @@
+migrator_ts: 1683090168
+__migrator:
+  kind:
+    version
+  use_local: true  # TODO: Remove this line after completing testing
```

Comment:
Adding to simplify testing. Though this needs to be dropped after. Flagging as a reminder

```suggestion
```